doc/visual-programming/source/widgets/evaluate/calibrationplot.md

```diff
@@ -7,21 +7,41 @@ Shows the match between classifiers' probability predictions and actual class pr
 
 - Evaluation Results: results of testing classification algorithms
 
-The [Calibration Plot](https://en.wikipedia.org/wiki/Calibration_curve)plots class probabilities against those predicted by the classifier(s).
+**Outputs**
 
-![](images/CalibrationPlot-stamped.png)
+- Calibrated Model: a model, calibrated with a user-set probability threshold
+
+The [Calibration Plot](https://en.wikipedia.org/wiki/Calibration_curve) plots class probabilities against those predicted by the classifier(s). We would use this widget to see whether a classifier is overly optimistic (gives predominantly positive results) or pessimistic (gives predominantly negative results). The widget can also output a calibrated model, where the user sets his/her own probability threshold.
+
+![](images/Calibration-Plot.png)
 
 1. Select the desired target class from the drop down menu.
-2. Choose which classifiers to plot. The diagonal represents optimal behavior; the closer the classifier's curve gets, the more accurate its prediction probabilities are. Thus we would use this widget to see whether a classifier is overly optimistic (gives predominantly positive results) or pessimistic (gives predominantly negative results).
-3. If *Show rug* is enabled, ticks are displayed at the bottom and the top of the graph, which represent negative and positive examples respectively. Their position corresponds to the classifier's probability prediction and the color shows the classifier. At the bottom of the graph, the points to the left are those which are (correctly) assigned a low probability of the target class, and those to the right are incorrectly assigned high probabilities. At the top of the graph, the instances to the right are correctly assigned high probabilities and vice versa.
-4. Press *Save Image* if you want to save the created image to your computer in a .svg or .png format.
-5. Produce a report.
+   - *Show rug*: If enabled, ticks are displayed at the bottom and the top of the graph, which represent negative and positive examples respectively. Their position corresponds to the classifier's probability prediction and the color shows the classifier. At the bottom of the graph, the points to the left are those which are (correctly) assigned a low probability of the target class, and those to the right are incorrectly assigned high probabilities. At the top of the graph, the instances to the right are correctly assigned high probabilities and vice versa.
```

Comment:
and the color shows the classifier -> and different colors represent different classifiers